lib/MQTTPlateRecorder.js

```diff
@@ -172,14 +183,16 @@ export default class MQTTPlateRecorder extends PlateRecorder {
 				`${this.baseTopic}/${source.name}/plate`,
 				result.plate
 			);
-			this.#client.publish(
-				`${this.baseTopic}/${source.name}/roi`,
-				jpeg
-			);
-			this.#client.publish(
-				`${this.baseTopic}/${source.name}/image`,
-				await original.toJpegBuffer()
-			);
+			if(this.enableImage || this.hassDiscovery.enable) {
```

Comment:
Please make enableImage (publishImage) be the only thing that controls whether an image is published or not. HassDiscovery likely needs to be updated to support "publishImage" itself.